schemas/data.schema.json

```
@@ -57,212 +114,39 @@
-                    "description": "Browser versions that most-recently introduced the feature",
-                    "properties": {
-                      "chrome": {
-                        "type": "string"
-                      },
-                      "chrome_android": {
-                        "type": "string"
-                      },
-                      "edge": {
-                        "type": "string"
-                      },
-                      "firefox": {
-                        "type": "string"
-                      },
-                      "firefox_android": {
-                        "type": "string"
-                      },
-                      "safari": {
-                        "type": "string"
-                      },
-                      "safari_ios": {
-                        "type": "string"
-                      }
-                    },
-                    "type": "object"
-                  }
-                },
-                "required": [
-                  "baseline",
-                  "support"
-                ],
-                "type": "object"
-              },
-              "description": "Statuses for each key in the feature's compat_features list, if applicable. Not available to the npm release of web-features.",
-              "type": "object"
-            },
-            "support": {
-              "additionalProperties": false,
-              "description": "Browser versions that most-recently introduced the feature",
-              "properties": {
-                "chrome": {
-                  "type": "string"
-                },
-                "chrome_android": {
-                  "type": "string"
-                },
-                "edge": {
-                  "type": "string"
-                },
-                "firefox": {
-                  "type": "string"
-                },
-                "firefox_android": {
-                  "type": "string"
-                },
-                "safari": {
-                  "type": "string"
-                },
-                "safari_ios": {
-                  "type": "string"
-                }
-              },
-              "type": "object"
-            }
-          },
-          "required": [
-            "baseline",
-            "support"
-          ],
-          "type": "object"
+          "description": "Whether a feature is considered a \"Baseline\" web platform feature and when it achieved that status",
+          "$ref": "#/definitions/StatusHeadline"
+        },
+        "discouraged": {
+          "description": "Whether developers are formally discouraged from using this feature",
+          "$ref": "#/definitions/Discouraged"
         }
       },
-      "required": [
-        "name",
-        "description",
-        "description_html",
-        "spec",
-        "status"
-      ],
-      "type": "object"
+      "required": ["name", "description", "description_html", "spec"]
```

Is status no longer required?
```suggestion
      "required": ["name", "description", "description_html", "spec", "status"]
```

---

Fixed in b2b6762.